docs/docs-ref-autogen/excel/excelscript/excelscript.cellvalueconditionalformat.yml

```diff
@@ -4,7 +4,40 @@ uid: 'ExcelScript!ExcelScript.CellValueConditionalFormat:interface'
 package: ExcelScript!
 fullName: ExcelScript.CellValueConditionalFormat
 summary: Represents a cell value conditional format.
-remarks: ''
+remarks: |-
+
+
+  #### Examples
+
+  ```TypeScript
+  /**
+   * This script applies conditional formatting to a range.
+   * That formatting is conditional upon the cell's numerical value.
+   * Any value between 50 and 75 will have the cell fill color changed and the font made italic.
+   */
+  function main(workbook: ExcelScript.Workbook) {
+    // Get the range to format.
+    const sheet = workbook.getActiveWorksheet();
+    const ratingColumn = sheet.getRange("D2:D20");
+
+    // Add cell value conditional formatting.
+    const cellValueConditionalFormatting : ExcelScript.CellValueConditionalFormat 
+      = ratingColumn.addConditionalFormat(ExcelScript.ConditionalFormatType.cellValue).getCellValue();
+    
+    // Set the format to apply when the condition is met.
+    let format = cellValueConditionalFormatting.getFormat();
+    format.getFill().setColor("yellow");
+    format.getFont().setItalic(true);
+
+    // Create the condition, in this case when the cell value is between 50 and 75.
```

Comment:
For this sample, why is the condition created after the format is applied? Since the condition is the purpose of the sample, I think it would make more sense to create the condition earlier, maybe: 

1. Get range 
2. Create condition 
... etc

Reply:
Both the rule and the format need to exist before conditional formatting can happen, but I agree that making the condition seems like the logical ordering.

Reply:
Instead of reordering, perhaps the comments could explicitly state this nuance.

Reply:
I think I'll leave it with the reordering and see if we get feedback or confusion. I think conditional formatting might be an area the maker-tier of Excel users are pretty familiar with.  The UI does do condition-then-formatting, so I'll match that:
![image](https://user-images.githubusercontent.com/38896772/178066579-046cbef5-d7b0-4a55-8a27-f9e6e276a39a.png)